tutorials/early_stopping/early_stopping.ipynb

```diff
@@ -0,0 +1,1067 @@
+       "19   0.9768            9      9_0            107             47   \n",
+       "1    0.9722           10     10_0             82             29   \n",
+       "2    0.9694           11     11_0            122             52   \n",
+       "4    0.9762           12     12_0            128             31   \n",
+       "5    0.9649           13     13_0            111             60   \n",
+       "6    0.9784           14     14_0            128             40   \n",
+       "7    0.9747           15     15_0            128             37   \n",
+       "8    0.9634           16     16_0             85             23   \n",
+       "9    0.9744           17     17_0             94             38   \n",
+       "10   0.9677           18     18_0            128             36   \n",
+       "11   0.9770           19     19_0            128             39   \n",
+       "\n",
+       "    learning_rate   dropout   trial_status generation_method  \n",
+       "0        0.000330  0.143252      COMPLETED             Sobol  \n",
+       "3        0.000859  0.200457      COMPLETED             Sobol  \n",
+       "12       0.000873  0.399055      COMPLETED             Sobol  \n",
+       "13       0.000218  0.419738  EARLY_STOPPED             Sobol  \n",
+       "14       0.004758  0.374596  EARLY_STOPPED             Sobol  \n",
+       "15       0.001009  0.077409      COMPLETED              GPEI  \n",
+       "16       0.000395  0.149083  EARLY_STOPPED              GPEI  \n",
+       "17       0.000879  0.165904      COMPLETED              GPEI  \n",
+       "18       0.000893  0.031481  EARLY_STOPPED              GPEI  \n",
+       "19       0.000889  0.150700      COMPLETED              GPEI  \n",
+       "1        0.000793  0.062578      COMPLETED              GPEI  \n",
+       "2        0.000781  0.005143      COMPLETED              GPEI  \n",
+       "4        0.000928  0.132889      COMPLETED              GPEI  \n",
+       "5        0.001189  0.142124  EARLY_STOPPED              GPEI  \n",
+       "6        0.000819  0.119113      COMPLETED              GPEI  \n",
+       "7        0.001162  0.163787      COMPLETED              GPEI  \n",
+       "8        0.002022  0.014043  EARLY_STOPPED              GPEI  \n",
+       "9        0.000794  0.133813      COMPLETED              GPEI  \n",
+       "10       0.001014  0.078035  EARLY_STOPPED              GPEI  \n",
+       "11       0.000650  0.149710      COMPLETED              GPEI  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "exp_to_df(experiment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ddc543de",
+   "metadata": {},
+   "source": [
+    "## Visualizations\n",
+    "\n",
+    "Finally, we show a visualization of learning curves versus actual elapsed wall time. This helps to illustrate that stopped trials make room for additional trials to be run."
```

Comment:
Maybe ccomment here on the fact that things are run in a fully async fashion with 3 trials running concurrently (and the gaps is where the candidate generation happens)>

Reply:
It would also be nice to discuss some of the key points here, e.g. 
- the fact that we're getting better over time, 
- that really bad trials are stopped very quickly , while others may run for longer
- what the tl;dr is and why we want this (saving compute and wall time; in general it would be nice to give an estimates of the savings in this example compared to the baseline of running everything until the end (for the purposes of this tutorial we can probably assume that we identify the same best performing trial that we would have had we run the full training for all trials.